vars/withInfraPipeline.groovy

```diff
@@ -37,7 +37,7 @@ def call(String product, String component = null, Closure body) {
       stageWithAgent('Checkout', product) {
         checkoutScm(pipelineCallbacksRunner: callbacksRunner)
       }
-
+      warnAboutRenovateConfig()
```

Comment:
couldn't we put this in checkoutScm? it seems like you've had to put this in a lot of places